### PR TITLE
Counter before start

### PR DESCRIPTION
**Pull request by nitro8708**

### CODE REVIEW

File: Home.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
```
Comment (KMConner) [inline Home.html:1]: This file was renamed to `index.html` in the commit 4bb0e3b076376ec303be6ee78b9f3bd9531288a4.
So you should edit `index.html` instead of adding this file.
(But the change to this file looks unnecessary...)

File: Loading.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```
Comment (KMConner) [inline Loading.html:1]: This file was also combined into `Gaming.html`, and not seemed to be changed.
So, this file should be deleted.

File: index.html [hunk 3]
```diff
@@ -1,109 +0,0 @@
-<!DOCTYPE html>
```
Comment (KMConner) [inline index.html:1]: Please do not delete this file...

File: HowToPlay.html [hunk 4]
```diff
@@ -63,16 +63,40 @@
 			font-size: 20px;
 			text-indent: 20px;
 			font-weight: bold;
+		}
```
Comment (KMConner) [inline HowToPlay.html:66]: Are the changes on this file necessary?

File: Gaming.html [hunk 5]
```diff
@@ -55,96 +55,118 @@
 /*			width: 600px;
 			height: 350px;*/
 			margin-top: 1%;
-			/* padding-bottom: 40%; */
-			width: 900px;
-			height: 750px;
+			padding-bottom: 40%;
+			width: 80%;
+			height: 0;
```
Comment (KMConner) [inline Gaming.html:60]: Are these changes necessary?

File: Home.html (hunk 1 above)
Comment (nitro8708) [reply Home.html:1]: yes, thanks for informing. 
There is no change on this file comparing with the prior version, so please instead it by the index.html, 
I will notice it the next time.

File: HowToPlay.html (hunk 4 above)
Comment (nitro8708) [reply HowToPlay.html:66]: I did not change this part in this push, but these lines are necessary.
There exactly some other changes in this HowToPlay.html just for test but not necessary, so please use the the prior version of this file, sorry for the mistakes.

File: Loading.html (hunk 2 above)
Comment (nitro8708) [reply Loading.html:1]: ok I see!

File: Gaming.html (hunk 5 above)
Comment (nitro8708) [reply Gaming.html:60]: This might be the conflicts with the newest-version code, so you can just instead them by the newest items, and just use the "Countdown" part.
